Dockerfile

```diff
@@ -9,72 +13,90 @@ ENV APP_HOME /src/cfgov-refresh
+ENV PIP_NO_CACHE_DIR true
+
+# Install python requirements
 COPY requirements requirements
 
-EXPOSE 8000
+RUN yum -y install gcc && \
+    pip install -r requirements/local.txt -r requirements/deployment.txt && \
+    yum -y remove gcc && \
+    yum clean all && rm -rf /var/cache/yum
 
-ENTRYPOINT ["./docker-entrypoint.sh"]
 
+EXPOSE 8000
 
-# Image designed for local developement
-FROM cfgov-base AS cfgov-develop
+ENTRYPOINT ["./docker-entrypoint.sh"]
+CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
 
-# Copy over the script that extends the Python environment with develop-apps
-COPY extend-environment.sh /etc/profile.d/extend-environment.sh
 
-# Install our requirements
-RUN pip install --no-cache-dir -r requirements/local.txt
 
-CMD ["python", "./cfgov/manage.py", "runserver", "0.0.0.0:8000"]
+FROM cfgov-dev as cfgov-prod
 
-FROM cfgov-develop as cfgov-build
+ENV SCL_HTTPD_VERSION httpd24
+ENV SCL_HTTPD_ROOT /opt/rh/${SCL_HTTPD_VERSION}/root
 
-# add node and yarn repos
-RUN curl -sL https://rpm.nodesource.com/setup_10.x | bash -
-RUN curl -sL https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo
+# Apache HTTPD settings
+#ENV APACHE_SERVER_ROOT ${SCL_HTTPD_ROOT}/etc/httpd
+ENV APACHE_SERVER_ROOT ${APP_HOME}/cfgov/apache
+ENV APACHE_PROCESS_COUNT 4
+ENV ACCESS_LOG /dev/stdout
+ENV ERROR_LOG /dev/stderr
+ENV STATIC_PATH ${APP_HOME}/cfgov/static/
 
-RUN yum install -y nodejs yarn  && \
-    yum clean all && rm -rf /var/cache/yum
+# mod_wsgi settings
+ENV CFGOV_PATH ${APP_HOME}
+ENV CFGOV_CURRENT ${APP_HOME}
+ENV PYTHONPATH ${APP_HOME}/cfgov
 
+# Django Settings
+ENV DJANGO_SETTINGS_MODULE cfgov.settings.production
+ENV DJANGO_STATIC_ROOT ${STATIC_PATH}
+ENV ALLOWED_HOSTS '["*"]'
 
-COPY cfgov/ ./cfgov
-COPY config/ ./config
-COPY gulp ./gulp
-COPY static.in ./static.in
-COPY scripts ./scripts
+RUN yum -y install ${SCL_HTTPD_VERSION} ${SCL_PYTHON_VERSION}-mod_wsgi && \
+    yum clean all && rm -rf /var/cache/yum && \
+    echo "source scl_source enable ${SCL_HTTPD_VERSION}" > /etc/profile.d/enable_scl_httpd.sh
 
-COPY docker-entrypoint.sh frontend.sh gulpfile.js jest.config.js package.json yarn.lock /src/cfgov-refresh/
+# See .dockerignore for details on which files are included
+COPY . .
 
-ENV DJANGO_SETTINGS_MODULE=cfgov.settings.production
-ENV DJANGO_STATIC_ROOT=/var/www/html/static
-ENV ALLOWED_HOSTS='["*"]'
+# Build frontend, cleanup excess file, and setup filesystem
+RUN yum -y install nodejs yarn  && \
+    ./frontend.sh production && \
+    cfgov/manage.py collectstatic && \
+    yarn cache clean && \
+    yum -y remove nodejs yarn && \
+    yum clean all && rm -rf /var/cache/yum && \
+    rm -rf \
+        cfgov/apache/www \
+        cfgov/unprocessed \
+        node_modules && \
+    ln -s ${SCL_HTTPD_ROOT}/etc/httpd/modules ${APACHE_SERVER_ROOT}/modules && \
+    mkdir -p cfgov/f/ && chown apache:apache cfgov/f/ && \
+    mkdir /tmp/eregs_cache && chown apache:apache /tmp/eregs_cache
```

Comment:
Request: comment here indicating that this is the local of the Django file-based cache?